pennylane/measurements/__init__.py

```diff
@@ -103,8 +103,10 @@ def circ(x, y):
 reset to the :math:`|0 \rangle` state by setting the ``reset`` keyword argument of ``qml.measure`` to ``True``.
 
 Users can also collect statistics on mid-circuit measurements along with other terminal measurements. Currently,
-``qml.expval``, ``qml.probs``, ``qml.sample``, ``qml.counts``, and ``qml.var`` are supported. Users have the
-ability to collect statistics on single measurement values.
+``qml.expval``, ``qml.probs``, ``qml.sample``, ``qml.counts``, and ``qml.var`` are supported. ``qml.probs``,
+``qml.sample``, and ``qml.counts`` support sequences of measurement values, ``qml.expval`` and ``qml.var`` do not.
+Statistics of arithmetic combinations of measurement values are supported by all but ``qml.probs``, and only as
+long as they are not collected in a sequence.
```

Comment:
> and only as long as they are not collected in a sequence.

Wait what does that mean?

Reply:
You can do arithmetic combinations (`m1 + m2`) and you can do a `Sequence` (`[m1, m2]` or `(m1, m2)`) but you can't do both (`[m1+m2, m3]`).

Reply:
Should I clarify this somehow? Do you have an idea for how this could be clearer?

Reply:
Ohhh! Ok understood. Maybe
> ... not collected in a sequence, e.g., `[m1+m2, m1-m2]`